pkgs/applications/audio/uhhyou.lv2/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, fetchgit, pkgconfig, libjack2, liblo, libGL, libX11, lv2, fftw }:
+
+stdenv.mkDerivation rec {
+  pname = "uhhyou.lv2";
```

Comment:
What's this specific `pname` has to do anything with the source?

Reply:
It's what he calls them himself:
https://github.com/ryukau/LV2Plugins#color-configuration
And also what the arch and sus pkgs are called:
https://github.com/ryukau/LV2Plugins#package

Reply:
Please put this in a comment.

Reply:
This review wasn't addressed as well.